pages/api/question.ts

```diff
@@ -86,3 +92,13 @@ export type UnitPriceModel = {
   singularFruit: string,
   pluralFruit: string,
 };
+
+  export type SalesTaxModel = {
+    price: number;
+    number: number;
+    taxRate: number;
+    personName: string;
+    image2: string;
```

Comment:
why image2?

Reply:
because they needed to be consistent throughout I thought?